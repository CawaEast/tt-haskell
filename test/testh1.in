let q = r in
let T = q q in \f.\x. f (f x)