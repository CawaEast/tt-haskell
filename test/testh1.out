((v8->v8)->(v8->v8))
T:v4
q:v1
r:v1
